common/app/model/CardStylePicker.scala

```diff
@@ -9,19 +9,23 @@ import commercial.targeting.CampaignAgent
 
 object CardStylePicker {
 
-  def apply(content: CapiContent, trailMetaData: MetaDataCommonFields): CardStyle = {
+  def apply(content: CapiContent): CardStyle = {
     val tags = content.tags.map(_.id).toSeq
     extractCampaigns(tags) match {
       case Nil => CardStyle(content, TrailMetaData.empty)
-      case _   => SpecialReport
+      case campaign :: _ =>
+        if (campaign.id.toString.toLowerCase() == "specialreportalt") SpecialReportAlt else SpecialReport
+      case _ => SpecialReport
```

Comment:
I'm not entirely sure I have a good grasp of what's happening here, but I think you can put the if statement inside the case?

```scala
case campain :: _ if campaign.id.toString.toLowerCase() == "specialreportalt" => SpecialReportAlt
case _ => SpecialReport
```